osmoutils/accum/accum_test.go

```diff
@@ -318,3 +321,439 @@ func (suite *AccumTestSuite) TestClaimRewards() {
 		})
 	}
 }
+
+func (suite *AccumTestSuite) TestAddToPosition() {
+	type testcase struct {
+		startingNumShares sdk.Dec
+		startingUnclaimedRewards sdk.DecCoins
+		newShares   sdk.Dec
+
+		// accumInit and accumDelta specify the initial accum value 
+		// and how much it has changed since the position being added 
+		// to was created
+		accumInit	sdk.DecCoins
+		accumDelta  sdk.DecCoins
+
+		// Address does not exist
+		addrDNE bool
```

Comment:
nit: consider expanding the abbreviation. Verbose identifiers are easier to review

Reply:
Done!